podcastit.md

```diff
@@ -1,47 +1,36 @@
 # Podcastit
 
-## Tietosurma
+## Tietotekniikka
```

Comment:
I think "Tietosurma" had been a pun on "Tietoturva", but considering only one of the podcasts here was netsec related, this'll be better :)